scripts/src/main/resources/scripts/environment-project.bat

```diff
@@ -84,29 +84,25 @@ set "line=%*"
 if "%~1%" == "export" (
   rem remove export as this only makes sense on linux/mac
   set line=!line:~7!
-  if "%~3%" == "" (
-    set "search=%~2%"
-  ) else (
-    set "search=%~2% "
-  )
-  set "replacement=%~2%="
+  shift
+)
+if "%~2%" == "" (
+  set "search=%~1%"
 ) else (
-  if "%~2%" == "" (
-    set "search=%~1%"
-  ) else (
-    set "search=%~1% "
-  )
-  set "replacement=%~1%="
+  set "search=%~1% "
+)
+set "replacement=%~1%="
+set value=%~2
```

Comment:
Nope. `%2` is the second argument and `%~2` expands it and unquotes.
https://ss64.com/nt/syntax-args.html

So your question might be why is there an extra percent sign after the first arg.
I can only reasonable answer for the `%~1%=` case as here the `%` sign is an escape for the equals sign.
In the other case (line 92) it may be obsolete to add the extra percent.
I can test that and can also not tell if I initially wrote this code as Rohit supported the fix #395

Reply:
So also the equals sign does not need to be escaped with percent sign.
To make you happy and avoid further confusion I removed the pointless percent signs.
According to my tests in Win VM it does not make any difference so fingers crossed that this will not break anything in this fragile CMD universe...
Can we resolve our conversations now and merge this PR?

Reply:
Well done to remove the trailing percent here. They should not be used after positional parameters.

Only variables within scripts must be surrounded by percents - for positional parameters this is just accepted for "convenience".  (The percent is no escape char here)
A "%1%2" still resolves to first pos. param followed by 2nd,

From today's view point it is easy to say, this was a bad design decision - but in the 80'ies this was not so easy to forsee.